Server/src/quadradiusr_server/config.py

```diff
@@ -5,16 +5,25 @@
 import toml as toml
 
 
+@dataclass
+class AuthConfig:
+    token_exp: int = 60
+    token_leeway: int = 10
```

Comment:
Leeway is a 'grace time' which allows for certain time variance. If the token expires at time T, the server will accept it event at T+10, because the client may have a bit different time set locally. The time is in seconds, but I'm not sure how I should signify it here. As a part of the property key? As a docstring?

Reply:
Asked out of curiosity, no need to add it here :D